<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>accountCreationScreen</class>
 <widget class="QDialog" name="accountCreationScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>440</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>565</width>
    <height>440</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Account Creater</string>
  </property>
  <property name="windowIcon">
   <iconset theme="SP_TitleBarMenuButton">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Account Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="editAccountName">
       <property name="text">
        <string>null</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Website</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4" colspan="2">
      <widget class="QLineEdit" name="editWebsite">
       <property name="text">
        <string>null</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Username</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="editUsername">
       <property name="text">
        <string>null</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4" colspan="2">
      <widget class="QLineEdit" name="editPassword">
       <property name="text">
        <string>null</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="editEmail">
       <property name="text">
        <string>null</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3" colspan="3">
      <widget class="QCheckBox" name="checkbox_Secret">
       <property name="text">
        <string>Does this account have secret questions?</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="3">
      <widget class="QLabel" name="label_5">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Secret Question 1</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QLabel" name="label_7">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Answer</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QLabel" name="label_8">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Answer</string>
       </property>
      </widget>
     </item>
     <item row="7" column="5">
      <widget class="QPushButton" name="btnSave">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3" colspan="2">
      <widget class="QLabel" name="label_6">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Secret Question 2</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Email</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3" colspan="3">
      <widget class="QLineEdit" name="editAnswer2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>null</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="3">
      <widget class="QLineEdit" name="editAnser1">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>null</string>
       </property>
      </widget>
     </item>
     <item row="7" column="4">
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="3">
      <widget class="QLineEdit" name="lineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>null</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3" colspan="3">
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>null</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
